pkgs/development/compilers/llvm/default.nix

```diff
@@ -56,8 +61,7 @@ let
         releaseInfo
         ;
       inherit (releaseInfo) release_version;
-      attrName =
-        args.name or (if (gitRelease != null) then "git" else lib.versions.major release_version);
```

Comment:
This change took me a few minutes to understand. Would be good to have a short explanation in the commit message.